lighthouse-core/lib/lh-error.js

```diff
@@ -242,6 +242,11 @@ const ERRORS = {
     message: UIStrings.badTraceRecording,
     lhrRuntimeError: true,
   },
+  NO_RESOURCE_SEND: {
```

Comment:
`NO_FIRST_RESOURCE`? `NO_FIRST_RESOURCE_SEND`?

Reply:
Does this need to be added to the proto, too, since it's a `lhrRuntimeError`? Again it's a bit premature, but we might easily forget to do it when the error is live.

Reply:
> Does this need to be added to the proto, too, since it's a lhrRuntimeError? Again it's a bit premature, but we might easily forget to do it when the error is live.

Yeah sg might as well do it now too.

re: `NO_FIRST_RESOURCE_SEND`, any particular reason you'd like to see the addition of `FIRST`? It seemed like unnecessarily specific information to add to me that might lead to narrower usage later. The only way there can be no first resource request is if there were none at all :)

Reply:
> re: `NO_FIRST_RESOURCE_SEND`, any particular reason you'd like to see the addition of `FIRST`? It seemed like unnecessarily specific information to add to me that might lead to narrower usage later. The only way there can be no first resource request is if there were none at all :)

it's just short and I was wondering if there's more context we can give it. That's a very good point, though :)

"Resource send" is kind of confusing as representing the _request_ for a resource, though (`ResourceSendRequest` does make sense with the other trace event names next to it). Maybe `NO_RESOURCE_REQUEST`? A slight step away from the trace event name but slightly more understandable on its own.

I'll stop bikeshedding after this :)

Reply:
> NO_RESOURCE_REQUEST

sold  :)